packages/terra-site/webpack.config.js

```diff
@@ -45,6 +51,7 @@ module.exports = {
                     'iOS >= 8',
                   ],
                 }),
+                CustomProperties(),
               ];
```

Comment:
Lets add it for our config. In our documentation on webpack config, I wouldn't include it, but add a section on how to add it. This way we can continue to test LTR/RTL styles but consumers have the choice to add the RTL plugin in their own webpack config if they need it.